src/staticOptions.ts

```diff
@@ -279,6 +282,7 @@ const staticOptions = asLanguageClientOptionsById({
   },
   postgresql: {
```

Comment:
`php` seems to be the only one without the `file` scheme

Reply:
fixed and amended